src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (b) => {
```

Comment:
The parameter name 'b' is not descriptive. Consider using a more descriptive name, such as 'value' or 'numberToAdd', to improve code readability.